http/readme.md

```diff
@@ -1,3 +1,42 @@
 # HyperText Transfer Protocol (HTTP)
 
+HTTP is a protocol **defines how client and server communicates**. 
+
+Everytime we search up something with our browser, let say Wikipedia, the browser will then communicate with Wikipedia's server to get the resources you had asked for.
+
+Let's see how the communication process works.
+  1. The **Browser send the request** for resources to the server. (in the form of URL)
+  2. The **Server process the request**, then **respond** accordingly to it.
+  
+### Examples
+**Request** and **Response** are the keywords in HTTP. To make it easier, imagine the communication between server and client as conversation between a Customer and a Bartender.
```

Comment:
Nice analogy! could we change it from bartender to maybe waiter? Keeping it friendly for potential young students.

Reply:
Sure! and Thank You!